build.sbt

```diff
@@ -2,19 +2,38 @@ inThisBuild(
   List(
     scalaVersion := "2.12.3",
     organization := "org.scalameta",
-    version := "0.1-SNAPSHOT",
-    sources.in(Compile, doc) := Nil // faster publishLocal.
+    sources.in(Compile, doc) := Nil, // faster publishLocal
+    licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    homepage := Some(url("https://github.com/scalameta/language-server")),
+    developers := List(
+      Developer("olafurpg", "Ólafur Páll Geirsson", "olafurpg@gmail.com", url("https://geirsson.com")),
+      Developer("gabro", "Gabriele Petronella", "gabriele@buildo.io", url("https://github.com/gabro")),
+      Developer("laughedelic", "Alexey Alekhin", "laughedelic@gmail.com", url("https://github.com/laughedelic"))
+    ),
+    scmInfo in ThisBuild := Some(
+      ScmInfo(
+        url("https://github.com/scalameta/language-server"),
+        s"scm:git:git@github.com:scalameta/language-server.git"
+      )
+    ),
+    releaseEarlyWith := BintrayPublisher,
+    releaseEarlyEnableSyncToMaven := false,
+    releaseEarlyEnableLocalReleases := true, // TODO: remove once CI publishing is setup
```

Comment:
This line can be removed if we don't want to publish manually

Reply:
I'd remove this. Manual publishing is bad.

Reply:
As long as `publishLocal` continues to work I'm happy ^^

Reply:
Yes, of course that works :smile:.

Reply:
Removed in 900a7e3